src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java

```diff
@@ -610,54 +610,24 @@ private String startBulkCaseBookPDFJob(String schema,
 										   String landscape) {
 									 	 
 
-		    String siteName = null;
-		    String studyName = null;
-		    String pdfHeader =  "StudyName: SiteName - Participant ParticipantID";
-		    
-		    Study site = null;
-		    Study study = null;
-		    if(siteOid !=null) {
-		    	site = studyDao.findByOcOID(siteOid.trim());
-		    	siteName = site.getName();
-		    }
-			if(studyOid != null) {
-				study = studyDao.findByOcOID(studyOid.trim());
-				studyName = study.getName();
-				
-				if(study.getStudy() != null) {
-					site = study;
-					study = study.getStudy(); 
-				}
-
-			}
+		    final	Study 	site = studyDao.findByOcOID(siteOid);	    			
```

Comment:
change to: final Study site = siteOid == null ? null : studyDao.findByOcOID(siteOid)

Reply:
done